cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PackCommand.java

```diff
@@ -177,6 +180,42 @@ public void execute() {
             if (buildOptions.dumpBuildTime()) {
                 BuildTime.getInstance().projectLoadDuration = System.currentTimeMillis() - start;
             }
+
+            Path readmeFilePath = null;
+
+            // 1. Check if a custom readme path is defined in Ballerina.toml
+            Optional<String> readmePathOptional = project.currentPackage().manifest().readme();
+            if (readmePathOptional.isPresent()) {
+                Path customPath = Paths.get(readmePathOptional.get());
+                readmeFilePath = customPath.isAbsolute()
+                        ? customPath
+                        : this.projectPath.resolve(customPath);
+            } else {
+                // 2. Default to README.md, then Package.md (for backward compatibility)
+                Path defaultReadme = this.projectPath.resolve("README.md");
+                Path legacyPackageMd = this.projectPath.resolve("Package.md");
+                if (Files.exists(defaultReadme)) {
+                    readmeFilePath = defaultReadme;
+                } else if (Files.exists(legacyPackageMd)) {
+                    readmeFilePath = legacyPackageMd;
+                }
+            }
+
+            // 3. Warn if the resolved readme file is missing or empty
+            if (readmeFilePath == null || !Files.exists(readmeFilePath)) {
+                CommandUtil.printError(this.errStream,"warning: Package.md is missing. " +
+                        "It is required when pushing to Ballerina Central.");
+            } else {
+                try {
+                    String content = Files.readString(readmeFilePath);
+                    if (content.trim().isEmpty()) {
+                        CommandUtil.printError(this.errStream, "WARNING: README.md file is missing in the package." +
+                                "Add meaningful documentation before pushing.");
```

Comment:
The compilation fails for the code because the method usage is incorrect. Please change it as suggested below.
```suggestion
                                "Add meaningful documentation before pushing.", null, false);
```